src/test.ml

```diff
@@ -142,7 +142,7 @@ let test_parsing = [
 let test_join_result_cols () =
   Tables.reset ();
   let ints = List.map (fun name ->
-    if String.ends_with name "?" then
+    if String.ends_with name ~suffix:"?" then
```

Comment:
especially confusing because `Stdlib.String` and `ExtString` have different argument order